test/test_type_hints.py

```diff
@@ -118,13 +119,17 @@ def test_doc_examples(self):
                 )
             except OSError:
                 raise unittest.SkipTest('cannot symlink') from None
-            (stdout, stderr, result) = mypy.api.run([
-                '--cache-dir=.mypy_cache/doc',
-                '--no-strict-optional',  # needed because of torch.lu_unpack, see gh-36584
-                os.path.abspath(fn),
-            ])
+            repo_rootdir = Path(__file__).resolve().parent.parent
+            # TODO: Would be better not to chdir here, this affects the
+            # entire process!
```

Comment:
could you please also include the github URL for the issue you created to discuss this?

Reply:
I didn't write this comment; it's from here: https://github.com/pytorch/pytorch/commit/8cd4dac78f32cb82e9a983e1d13006250ba78223#diff-a0c9bbf5bd1854c65f36d0a26a1a74c1bb0e0d0fa8c77d6573e9531bd310721eL166-L167